numerics/чебышёв_series.hpp

```diff
@@ -0,0 +1,41 @@
+﻿#pragma once
+
+#include <vector>
+
+#include "geometry/named_quantities.hpp"
+#include "quantities/quantities.hpp"
+
+// Spelling: Чебышёв ЧЕБЫШЁВ чебышёв
```

Comment:
No, please, let's keep it here, it's a pain to have to look it up and do the case conversion.